src/app.html

```diff
@@ -7,6 +7,6 @@
 		%sveltekit.head%
 	</head>
 	<body data-sveltekit-preload-data="hover">
-		<div style="display: contents">%sveltekit.body%</div>
```

Comment:
According to MDN this is not advised. Was there a reason for this @erh ?

Reply:
no clue, just think i copied this from somewhere